server/writeModel/geocaching/cache.js

```diff
@@ -1,145 +1,200 @@
 'use strict';
 
+const { only } = require('wolkenkit-command-tools');
+const onlyIfCacheHasBeenPublished = require('../../shared/middleware/onlyIfCacheHasBeenPublished');
+const onlyIfCacheHasNotBeenPublished = require('../../shared/middleware/onlyIfCacheHasNotBeenPublished');
+
 const initialState = {
-	name: '',
-	description: '',
-	coordinate: {lat: 0, lon: 0},
-	countFavorites: 0,
-	published: false,
-	countFindings: 0,
-	comments: [],
-	removed: false,
-
-	isAuthorized: {
-		commands: {
-			publish: { forAuthenticated: true, forPublic: false },
-			favor: { forAuthenticated: true, forPublic: false },
-			find: { forAuthenticated: true, forPublic: false },
-			comment: { forAuthenticated: true, forPublic: false }
-		},
-		events: {
-			published: { forAuthenticated: true, forPublic: false },
-			favored: { forAuthenticated: true, forPublic: false },
-			found: { forAuthenticated: true, forPublic: false },
-			commented: { forAuthenticated: true, forPublic: false }
-		}
-	}
+  name: '',
+  description: '',
+  coordinate: { lat: 0, lon: 0 },
+  countFavorites: 0,
+  published: false,
+  countFindings: 0,
+  comments: [],
+  removed: false,
+
+  isAuthorized: {
+    commands: {
+      comment: { forAuthenticated: true, forPublic: false },
```

Comment:
You have changed this to alphabetical order. I personally would prefer the logical order. But what I do like is that you added the commands and events that are only visible to the owner.

Reply:
Makes sense, especially relating to the results of the event storming session it's easier finding the code for the event/command in the geocaching process.